Xcode/Configs/Common.xcconfig

```diff
@@ -10,6 +10,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "Version.xcconfig"
```

Comment:
Perhaps rebase on master?  This line shouldn't be necessary and it would be nice to be sure it doesn't merge oddly.